### DISCUSSION TITLE
best github or linkedln for portofolio

### DISCUSSION THREAD

**Discussion by ridhoreynaldo** [Schweinepriester/github-profile-achievements#245] [Q&A]
best github or linkedln for portofolio

**Comment by Salemzinho** [comment]
Linkedin as an academic and professional curriculum.

Github is best as a project portfolio.

But between the two, it is better to create a website to showcase your talents and projects.

**Comment by melvincwng** [comment]
@ridhoreynaldo, in my opinion, GitHub is the better option to be used as your portfolio.

Recruiters would be able to see your projects and the source code easily on GitHub.

**Reply by abhinit21** [reply]
Agree 👍

You can create a website on GitHub itself it will act for both purposes.
Redirect to GiHub on LinkedIn too

**Comment by MitkoVtori** [comment]
GitHub is much better for portfolio in every way. 
**You have access to many code tools, readme bio, complete list of activity, achievements and contributions, your own code repos & much others**

**Comment by Preetiraj3697** [comment]
GitHut is much better for portfolio in every way.
you use your portfolio in github page @ridhoreynaldo ridhoreynaldo.github.io.
